src/index.ts

```diff
@@ -112,12 +113,17 @@ export default function openAboutWindow(info: AboutWindowInfo) {
         window.webContents.send('about-window:info', info);
         if (info.open_devtools) {
             window.webContents.openDevTools({detach: true});
+            // window.webContents.openDevTools({mode: 'detach'}); // For Electron 2.0
```

Comment:
I will check that, because the build process crashed when I've applied the code commented.